package com.tournament.tournament_management_system.dto;

import com.tournament.tournament_management_system.model.RegistrationStatus;

import java.time.LocalDateTime;

public class TournamentParticipantResponse {

    private Long tournamentId;
    private Long participantId;
    private LocalDateTime registeredAt;
    private RegistrationStatus status;

    public Long getTournamentId() {
        return tournamentId;
    }

    public void setTournamentId(Long tournamentId) {
        this.tournamentId = tournamentId;
    }

    public Long getParticipantId() {
        return participantId;
    }

    public void setParticipantId(Long participantId) {
        this.participantId = participantId;
    }

    public LocalDateTime getRegisteredAt() {
        return registeredAt;
    }

    public void setRegisteredAt(LocalDateTime registeredAt) {
        this.registeredAt = registeredAt;
    }

    public RegistrationStatus getStatus() {
        return status;
    }

    public void setStatus(RegistrationStatus status) {
        this.status = status;
    }
}